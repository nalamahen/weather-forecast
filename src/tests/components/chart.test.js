import { shallow } from 'enzyme';
import React from 'react';
import Chart from '../../components/chart';

it('expect to number of component', () => {
    expect(shallow(<Chart />).length).toEqual(1);
});

it('expect to render chart component', () => {
    expect(shallow(<Chart />)).toMatchSnapshot();
});

