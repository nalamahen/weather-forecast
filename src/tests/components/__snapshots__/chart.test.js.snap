// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render chart component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <_default />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Sparklines
        data={Array []}
        height={100}
        margin={2}
        preserveAspectRatio="none"
        width={150}
      >
        <SparklinesLine
          onMouseMove={[Function]}
          style={Object {}}
        />
      </Sparklines>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <SparklinesLine
          onMouseMove={[Function]}
          style={Object {}}
        />,
        "data": Array [],
        "height": 100,
        "margin": 2,
        "preserveAspectRatio": "none",
        "width": 150,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "color": undefined,
          "onMouseMove": [Function],
          "style": Object {},
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Sparklines
          data={Array []}
          height={100}
          margin={2}
          preserveAspectRatio="none"
          width={150}
        >
          <SparklinesLine
            onMouseMove={[Function]}
            style={Object {}}
          />
        </Sparklines>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <SparklinesLine
            onMouseMove={[Function]}
            style={Object {}}
          />,
          "data": Array [],
          "height": 100,
          "margin": 2,
          "preserveAspectRatio": "none",
          "width": 150,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "color": undefined,
            "onMouseMove": [Function],
            "style": Object {},
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
